javascript-practice/src/js/helpers/helpers.js

```diff
@@ -62,3 +63,19 @@ export const convertDataObjectToModel = (data) => {
 
   return { id, ...object.data };
 };
+
+export const getSubdirectoryURL = () => {
+  const url = window.location.href;
+
+  const parts = url.split('/'); // Results: ['http:', '', 'example.com', '']
+
+  const subDirectory = parts[3]; // Get subdirectory url only
+
+  // Remove query behind subDirectory
+  const index = subDirectory.indexOf('?');
```

Comment:
After const/let block, should add empty line

Reply:
Fixed.